src/main/scala/io/iohk/ethereum/jsonrpc/EthMiningService.scala

```diff
@@ -101,7 +101,7 @@ class EthMiningService(
                 dagSeed = EthashUtils
                   .seed(
                     pb.block.header.number.toLong,
-                    node.blockchainConfig.forkBlockNumbers.ecip1099BlockNumber.toLong
+                    blockchainConfig.forkBlockNumbers.ecip1099BlockNumber.toLong
```

Comment:
In this edit, the construct "import node._," followed by directly accessing the inner def blockchainConfig is used here.
However, previous commit would directly refering to the complete node.blockchainConfig entry.

Is there a reason ?

Reply:
Yes, I am looking for ways to refactor these class structures in a way that minimizes changes we have to make. Importing something that provides the implicit parameters helps accomplish this. And in this case we can use it to access `blockchainConfig` this way.